specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -566,49 +566,16 @@
         }
       }
     },
-	"/skillsets('{skillsetName}')": {
-      "get": {
```

Comment:
Please re-order content in separate commits to make it easier to diff.

Reply:
Do take a look at the new commits.. they should hopefully be more self-evident